test/alternator/test_return_consumedcapacity.py

```diff
@@ -71,4 +80,125 @@ def test_long_get(test_table):
+        consumed_capacity = response.get('ConsumedCapacity')
+        assert 1 == consumed_capacity["CapacityUnits"]
+
+        response = table.put_item(Item={'p123': p, 'c4567': c, 'att': val, 'another': val2 + 'a'}, ReturnConsumedCapacity='TOTAL')  # val2 now passes 1KB
+        assert 'ConsumedCapacity' in response
+        consumed_capacity = response.get('ConsumedCapacity')
+        assert 2 == consumed_capacity["CapacityUnits"]
+
+
+def test_simple_put_item_bytes(test_table_sb):
+    p = random_string()
+    val = random_string()
+    c = random_bytes()
+    total_length = len(p) + len(val) + len(c) + len("pcattanother")
+
+    val2 = 'a' * (KB - total_length)  # val2 length pass the total length to KB
+    response = test_table_sb.put_item(Item={'p': p, 'c': c, 'att': val, 'another': val2}, ReturnConsumedCapacity='TOTAL')
+    assert 'ConsumedCapacity' in response
+    consumed_capacity = response.get('ConsumedCapacity')
+    assert 1 == consumed_capacity["CapacityUnits"]
+    response = test_table_sb.put_item(Item={'p': p, 'c': c, 'att': val, 'another': val2 + 'a'}, ReturnConsumedCapacity='TOTAL')
+    assert 'ConsumedCapacity' in response
+    consumed_capacity = response.get('ConsumedCapacity')
+    assert 2 == consumed_capacity["CapacityUnits"]
+
+
+def test_simple_get_item_bytes(test_table_sb):
+    p = random_string()
+    val = random_string()
+    c = random_bytes()
+    total_length = len(p) + len(val) + len(c) + len("pcattanother")
+
+    val2 = 'a' * (4 * KB - total_length)  # val2 length pass the total length to KB
+    test_table_sb.put_item(Item={'p': p, 'c': c, 'att': val, 'another': val2}, ReturnConsumedCapacity='TOTAL')
+    response = test_table_sb.get_item(Key={'p': p, 'c': c}, ConsistentRead=True, ReturnConsumedCapacity='TOTAL')
+    assert 'ConsumedCapacity' in response
+    consumed_capacity = response.get('ConsumedCapacity')
+    assert 1 == consumed_capacity["CapacityUnits"]
+    test_table_sb.put_item(Item={'p': p, 'c': c, 'att': val, 'another': val2 + 'a'}, ReturnConsumedCapacity='TOTAL')
+    response = test_table_sb.get_item(Key={'p': p, 'c': c}, ConsistentRead=True, ReturnConsumedCapacity='TOTAL')
+    assert 'ConsumedCapacity' in response
+    consumed_capacity = response.get('ConsumedCapacity')
+    assert 2 == consumed_capacity["CapacityUnits"]
+
+def num_length(n):
+    s = str(n)
+    l = len(s)
+    if '-' in s:
+        l += 1
+    l += 1
+    l /= 2
+    l += 1
+    return int(l)
+
+def test_number_magnitude_key(test_table_sn):
+    p = random_string()
+    # Legal magnitudes are allowed:
+    for num in [Decimal("3.14"),
+                Decimal("3"),
+                Decimal("3143846.26433832795028841"),
+                Decimal("31415926535897932384626433832795028841e30")]:
+        for sign in [False, True]:
+            if sign:
+                num = -num
+            x = random_string()
+            total_length = len(p) + len(x) + num_length(num) + len("pcaval2")+20
```

Comment:
You need to add a comment about this 20.  If I understand correctly, you said the number sizes are different in Scylla and DynamoDB. But why is the slack so huge? Do you also have an uncertainty of 20 bytes in the size of "3"?